lib/data_update_scripts/20211104200856_update_each_articles_privileged_user_point_count.rb

```diff
@@ -0,0 +1,13 @@
+module DataUpdateScripts
+  class UpdateEachArticlesPrivilegedUserPointCount
+    def run
+      # This code is "idempotent" in that it's calculating and caching a value stored elsewhere.
```

Comment:
agree - safe to run more than once (worst  case scenario is the same articles have their sums recalculated)